### PR TITLE
Final changes for accessibility

### PR DESCRIPTION
**Pull request by BillWagner**
I am less than thrilled with the Get Started section. This is a temporary fix until more of the re-work of the C# guide takes place.

### CODE REVIEW

File: docs/csharp/properties.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ the *null conditional operator*. It checks for a null reference before evaluatin
 the right side of the operator. The end result is that if there are no subscribers
 to the `PropertyChanged` event, the code to raise the event doesn't execute. It would
 throw a `NullReferenceException` without this check in that case. For more information,
-see [`events`](delegates-events.md). This example also uses the new
+see [`events`](delegates-overview.md). This example also uses the new
```
Comment (mairaw) [inline docs/csharp/properties.md]: Should it go here instead?
```suggestion
see [`events`](events-overview.md). This example also uses the new
```

File: docs/toc.yml [hunk 2]
```diff
@@ -501,11 +501,13 @@
   - name: Get Started
     href: csharp/getting-started/
   - name: Tutorials
-    href: csharp/tutorials/index.md
     items:
-    - name: Introduction to C#
-      href: csharp/tutorials/intro-to-csharp/index.md
+    - name: Index of tutorials
```
Comment (mairaw) [inline docs/toc.yml]: Shouldn't we stick with overview in these cases?

File: docs/toc.yml [hunk 3]
```diff
@@ -664,8 +667,9 @@
     - name: Versioning
       href: csharp/versioning.md
   - name: How To C# Topics
-    href: csharp/how-to/index.md
     items:
+    - name: Topic index
```
Comment (mairaw) [inline docs/toc.yml]: Need to think about this one...

**Review by mairaw — APPROVED** [review]
This looks good @BillWagner. Need some time to think about some options. Some comments for your consideration.

File: docs/csharp/getting-started/toc.yml [hunk 4]
```diff
@@ -1,5 +1,6 @@
 - name: Getting Started
-  href: index.md
   items:
+  - name: How to use this guide
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/csharp/getting-started/toc.yml]: I agree. This is on my list (early next week) once we merge these three related PRs.

The right fix here moves a number of other elements: The introductory tutorials should be part of "Get started".

File: docs/toc.yml (hunk 2 above)
Comment (BillWagner) [reply docs/toc.yml]: I think you're right.

File: docs/toc.yml (hunk 3 above)
Comment (BillWagner) [reply docs/toc.yml]: I agree. this one is another short-term work around to make it a bit better before doing the long term fix.

The long term fix as part of the content model updates (see #11606) is to move the "How to" articles further down-level in the TOC: The "strings" how to articles with conceptual content on strings, how to LINQ content with LINQ content and so on. This page would go away under that format. Until then, it's useful as a roadmap to all the "how to" content distributed throughout the guide.

**Pull request merged**